pkg/ocm/helpers.go

```diff
@@ -937,11 +937,20 @@ func ParseDiskSizeToGigibyte(size string) (int, error) {
 		return 0, fmt.Errorf("invalid disk size: %w", err)
 	}
```

Comment:
@thomasmckay Can you also add the `'` to these as well please so that we're consistent throughout the function?

cheers,

Rob

Reply:
I think I got them all now